docs/source/hyper-parameter-search.rst

```diff
@@ -465,6 +465,7 @@ to use post-estimation features like scoring or prediction, we recommend using
 :class:`dask_ml.wrappers.ParallelPostFit`.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
Can you update these to provide `meta=` whatever is appropraite?

Reply:
Thanks a lot for the review. Will add that.

Reply:
Added with commit https://github.com/dask/dask-ml/pull/862/commits/094c31adc3352c50966f86d501739b1ea50a9c9e